appservice/src/editScmType.ts

```diff
@@ -11,10 +11,11 @@ import { connectToGitHub } from './github/connectToGitHub';
 import { localize } from './localize';
 import { ScmType } from './ScmType';
 import { SiteClient } from './SiteClient';
+import { DeploymentsTreeItem } from './tree/DeploymentsTreeItem';
 import { ISiteTreeRoot } from './tree/ISiteTreeRoot';
 import { nonNullProp } from './utils/nonNull';
 
-export async function editScmType(client: SiteClient, node: AzureTreeItem<ISiteTreeRoot>, context: IActionContext, newScmType?: ScmType, showToast: boolean = true): Promise<ScmType | undefined> {
+export async function editScmType(client: SiteClient, node: AzureTreeItem<ISiteTreeRoot> | DeploymentsTreeItem, context: IActionContext, newScmType?: ScmType, showToast: boolean = true): Promise<ScmType | undefined> {
```

Comment:
We don't _really_ need the node. We just need `ISubscriptionContext` for `openInPortal` down the line. That would kind of separate out the tree stuff from this function